typhos/positioner.py

```diff
@@ -1009,32 +1024,20 @@ def _create_signal_panel(self) -> Optional[TyphosSignalPanel]:
         if self.device is None:
             return None
 
-        panel = TyphosSignalPanel()
-        panel.omitNames = self.get_names_to_omit()
-        panel.sortBy = SignalOrder.byName
-        panel.add_device(self.device)
-
-        self.ui.layout().addWidget(panel)
-        return panel
+        return RowDetails(row=self)
```

Comment:
```suggestion
        return RowDetails(row=self, parent=self, flags=QtCore.Qt.Window)
```

If you pass the flags kwarg through to QWidget, the little window will close when the parent closes but still show up as a separate window

```python
class RowDetails(QtWidgets.QWidget):
    """
    Container class for floating window with positioner row's basic config info.
    """
    row: TyphosPositionerRowWidget
    resize_timer: QtCore.QTimer

    def __init__(self, row: TyphosPositionerRowWidget, parent: QtWidgets.QWidget | None = None, **kwargs):
        super().__init__(parent=parent, **kwargs)
        self.row = row
```

Reply:
Thank you, I didn't know this at all